doc/man3/OSSL_PARAM_allocate_from_text.pod

```diff
@@ -28,28 +28,21 @@ size (see L<OSSL_PARAM(3)> for more information).
 
 OSSL_PARAM_allocate_from_text() takes a control I<key>, I<value> and
 value size I<value_n>, and given a parameter descriptor array
-I<paramdefs>, it converts the value to something suitable for
-L<OSSL_PARAM(3)> and stores that in the buffer I<buf>, and modifies
-the parameter I<to> to match.
-I<buf_n>, if not NULL, will be assigned the number of bytes used in
-I<buf>.
-If I<buf> is NULL, only I<buf_n> will be modified, everything else is
-left untouched, allowing a caller to find out how large the buffer
-should be.
-I<buf> needs to be correctly aligned for the type of the B<OSSL_PARAM>
-I<key>.
-If <found> is not NULL, it is set to 1 if the parameter can be located and
-to 0 otherwise.
-
-The caller must remember to free the data of I<to> when it's not
-useful any more.
+I<paramdefs>, it converts the value to something suitable for an
+L<OSSL_PARAM(3)> item, and stores the result in I<to>.
+
+The I<data> field in I<to> allocated using L<OPENSSL_zalloc(3)> and
+needs to be freed by the caller when it's not useful any omre, using
```

Comment:
anymore, 1 word

Reply:
Could also be 2 words, looks like anymore in 1 word might be used by some US people, and looks like everybody might be happy with the 2 words.

Reply:
`omre` should be `more`

Reply:
`omre` fixed